lib/l10n/app_fr.arb

```
@@ -3459,10 +3461,59 @@
+  "@trashCount": {
+    "placeholders": {
+      "count": {
+        "type": "int"
+      }
+    }
+  },
+  "trashRetentionPeriod": "Période de rétention de la corbeille",
+  "trashRetentionOption7Days": "7 jours",
+  "trashRetentionOption30Days": "30 jours",
+  "trashRetentionOption90Days": "90 jours"
+  ,"insertImage": "Insérer une image",
```

RuboCop (1.86.1)

[fatal] unexpected token tCOMMA
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)